src/org/labkey/targetedms/parser/SkylineDocumentParser.java

```diff
@@ -2200,29 +2200,38 @@ private void readGeneralTransition(XMLStreamReader reader, GeneralTransition tra
         if(explicitCollisionEnergy != null)
             transition.setExplicitCollisionEnergy(Double.valueOf(explicitCollisionEnergy));
 
-        String sLens = reader.getAttributeValue(null, "s_lens");
+        String sLens = reader.getAttributeValue(null, "explicit_s_lens");
+        if (sLens == null)
+            // Fall back on older attribute name
+            sLens = reader.getAttributeValue(null, "s_lens");
         if(sLens != null)
-            transition.setExplicitCollisionEnergy(Double.valueOf(sLens));
+            transition.setsLens(Double.valueOf(sLens));
 
-        String coneVoltage = reader.getAttributeValue(null, "cone_voltage");
+        String coneVoltage = reader.getAttributeValue(null, "explicit_cone_voltage");
+        if (coneVoltage == null)
+            coneVoltage = reader.getAttributeValue(null, "cone_voltage");
         if(coneVoltage != null)
-            transition.setExplicitCollisionEnergy(Double.valueOf(coneVoltage));
+            transition.setConeVoltage(Double.valueOf(coneVoltage));
 
         String explicitCompensationVoltage = reader.getAttributeValue(null, "explicit_compensation_voltage");
         if(explicitCompensationVoltage != null)
-            transition.setExplicitCollisionEnergy(Double.valueOf(explicitCompensationVoltage));
+            transition.setExplicitCompensationVoltage(Double.valueOf(explicitCompensationVoltage));
 
         String explicitDeclusteringPotential = reader.getAttributeValue(null, "explicit_declustering_potential");
         if(explicitDeclusteringPotential != null)
-            transition.setExplicitCollisionEnergy(Double.valueOf(explicitDeclusteringPotential));
+            transition.setExplicitDeclusteringPotential(Double.valueOf(explicitDeclusteringPotential));
 
-        String explicitDriftTimeMsec = reader.getAttributeValue(null, "explicit_drift_time_msec");
+        String explicitDriftTimeMsec = reader.getAttributeValue(null, "explicit_ion_mobility_high_energy_offset");
+        if (explicitDriftTimeMsec == null)
+            explicitDriftTimeMsec = reader.getAttributeValue(null, "explicit_drift_time_msec");
```

Comment:
"explicit_drift_time_msec" is not at the transition level.  It used to be at the precursor level and was renamed to "explicit_ion_mobility" there.  So it does not make sense reading this for a transition.  explicitDriftTimeMsec column can be deleted from GeneralTransition in 20.1.

Reply:
OK, deleted with latest push.